python/cuml/internals/CMakeLists.txt

```diff
@@ -13,13 +13,14 @@
 # =============================================================================
 
 
-
 set(cython_sources
-    base.pyx
-    internals.pyx
     logger.pyx
 )
 
+set(cython_sources "")
+add_module_gpu_default("base.pyx")
```

Comment:
Don't these two pyx files need to be added to the CPU build as well?

Reply:
Ah yes I think so, totally messed that up in the update of location of files